README.md

```diff
@@ -6,220 +6,17 @@
 
 <img src="https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/docs/cellxgene-demo-1.gif" width="200" height="200" hspace="30"><img src="https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/docs/cellxgene-demo-2.gif" width="200" height="200" hspace="30"><img src="https://raw.githubusercontent.com/chanzuckerberg/cellxgene/master/docs/cellxgene-demo-3.gif" width="200" height="200" hspace="30">
 
-## getting started
+Want to see where we are going? Check out [our roadmap](ROADMAP.md).
 
-You'll need **python 3.6+** and **Google Chrome**.
-The web UI is tested on OSX and Windows using Chrome, and the python CLI is tested on OSX and Ubuntu (via WSL/Windows). It should work on other platforms, but if you run into trouble let us know (see [help](#help-and-contact) below).
+## Documentation
```

Comment:
We had been using all lowercase for headings to match with the all lowercase cellxgene. I personally liked this, but  understand if we want to go back to a more standard style guide.

Reply:
I would suggest adding the lowercase headings to the website too if we decide to keep this direction.

Reply:
ah, gotcha